right-click-for-superpowers/script.js

```diff
@@ -0,0 +1,117 @@
+import {FilesetResolver, LlmInference} from 'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-genai';
```

Comment:
Please add license header to all files containing code. E.g. https://github.com/GoogleChromeLabs/web-ai-demos/blob/main/product-reviews/src/.consts.js#L1

Reply:
Done